kyuubi-server/src/main/scala/org/apache/kyuubi/server/metadata/jdbc/JDBCMetadataStore.scala

```diff
@@ -78,17 +79,17 @@ class JDBCMetadataStore(conf: KyuubiConf) extends MetadataStore with Logging {
 
   private def initSchema(): Unit = {
     val classLoader = Utils.getContextOrKyuubiClassLoader
-    val initSchemaStream: Option[InputStream] = dbType match {
-      case DERBY =>
-        Option(classLoader.getResourceAsStream("sql/derby/metadata-store-schema-derby.sql"))
-      case MYSQL =>
-        Option(classLoader.getResourceAsStream("sql/mysql/metadata-store-schema-mysql.sql"))
+    val initSchemaFile: Option[File] = dbType match {
+      case DERBY | MYSQL =>
+        getInitSchemaStream(classLoader.getResource(s"sql/${dbType.toString.toLowerCase}"))
       case CUSTOM => None
     }
-    initSchemaStream.foreach { inputStream =>
+    initSchemaFile.foreach { schemaFile =>
+      info(s"Init schema file: ${schemaFile.getAbsolutePath}")
+      val reader = new BufferedReader(new FileReader(schemaFile))
```

Comment:
will raise a pr to followup it, we have to use inputStream instead of FileReader

Reply:
FYI:

https://github.com/lightning-L/incubator-kyuubi/pull/1/files


You can use FileReader for UT, but for binary, there is no File, there is just Resource.

Reply:
this is another follow up: https://github.com/apache/kyuubi/pull/4103

you can merge the https://github.com/lightning-L/incubator-kyuubi/pull/2/files

Reply:
I refer this: https://stackoverflow.com/questions/1429172/how-to-list-the-files-inside-a-jar-file/28057735#28057735

Reply:
the UT in https://github.com/apache/kyuubi/pull/4103 has passed. @lightning-L